Entities/TDExplosive.cpp

```diff
@@ -63,7 +63,10 @@ namespace RTE {
 		ThrownDevice::Update();
 
 		if (m_Activated) {
-			m_SpriteAnimMode = !m_IsAnimatedManually ? ALWAYSLOOP : NOANIM;
```

Comment:
This probably warrants a changelog entry, since it'll affect modders' grenades pretty significantly.